src/App.vue

```diff
@@ -759,8 +759,8 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     // Prompt app validations.
     this.setAppValidate(true)
 
-    // Evaluate valid flags. Scroll to invalid components or file alteration.
-    if (await this.validateAndScroll(this.getFlagsReviewCertify, ComponentsReviewCertify)) {
+    // Evaluate valid flags. Scroll to invalid components or file alteration. Wait to allow app validation.
+    if (await this.$nextTick.validateAndScroll(this.getFlagsReviewCertify, ComponentsReviewCertify)) {
```

Comment:
The validations checks were happening before court order could be set invalid.

Reply:
Please make this 2 separate statements. (Maybe move `await Vue.nextTick()` right after setAppValidate above.)